tests/Forc.toml

```diff
@@ -10,19 +10,21 @@ members = [
   "./src/signed_integers/signed_i64",
   "./src/signed_integers/signed_i128",
   "./src/signed_integers/signed_i256",
+  "./src/signed_integers/signed_i8_twos_complement",
+  "./src/signed_integers/signed_i16_twos_complement",
+  "./src/signed_integers/signed_i32_twos_complement",
+  "./src/signed_integers/signed_i64_twos_complement",
```

Comment:
Looks like this doesn't include 128 and 256

Reply:
Yes, I'll add them in a smaller PR